requirements.txt

```diff
@@ -27,7 +27,7 @@ pandas==1.4.3
 passlib[bcrypt]==1.7.4
 plotly==5.4
 psycopg2-binary==2.9.3
-pydantic<1.10.0
+pydantic<1.10.2
```

Comment:
is there an actual fix here or just got bumped in the attempt to figure out the bug?

Either way it's fine, I'm just curious!

Reply:
I was trying to solve the validator reuse error in CI (this is going to be the theme to everything here) and I saw an older Pydantic issue where this error started happening after a python upgrade. They got the issue fixed, so I figured it was a long shot but worth a try since the issue had been closed for a long time. Turned out to make no difference, but since everything else was still working I figured I may as well leave the newer version in.